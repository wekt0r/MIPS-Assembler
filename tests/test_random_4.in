ori $s4, $s0, 185
srlv $9, $t3, $14
srl $t0, $8, 28
sub $k0, $t2, $s6
sra $s7, $6, 13
subu $12, $24, $15
sub $8, $5, $k0
mfhi $29
slt $s4, $t6, $t8
sll $t0, $20, 0
addu $25, $5, $3
addiu $t2, $fp, 26128
slti $15, $gp, 27137
mfhi $22
nor $7, $a0, $8
srav $21, $6, $s3
sltiu $25, $t6, 7909
and $3, $s1, $s6
subu $2, $v0, $13
mfhi $30
sllv $t1, $t8, $at
ori $k0, $s0, 29028
xori $30, $v0, 16312
srlv $18, $t2, $24
srl $9, $t5, 11
ori $15, $at, 17130
lui $t2, 13496
srl $20, $20, 5
add $t8, $ra, $gp
sll $k1, $16, 1
addu $29, $8, $t4
sltu $ra, $11, $27
srav $ra, $k0, $at
sra $t8, $s5, 5
divu $23, $2
mflo $s5
multu $v0, $17
slti $28, $5, 1769
ori $gp, $20, 7605
srav $k1, $t8, $t0
xor $v0, $s0, $t7
mflo $t7
slt $18, $s5, $0
addiu $21, $15, 21784
divu $k0, $25
srav $k1, $at, $6
srav $t9, $t7, $t8
sra $6, $0, 21
slt $9, $31, $4
mult $a3, $26
addiu $v0, $t5, 24440
subu $13, $26, $t5
mflo $ra
slt $18, $t2, $6
andi $10, $13, 6294
sll $s1, $5, 23
srlv $28, $28, $v0
srl $0, $v1, 5
addu $t8, $8, $v0
multu $18, $7
mfhi $s3
slt $21, $2, $a0
sub $t3, $29, $gp
mult $t1, $14
sll $11, $26, 6
addiu $14, $s7, 5431
add $t0, $t1, $k0
srlv $28, $a1, $s1
multu $19, $t8
mflo $gp
nor $14, $6, $at
lui $v0, 11978
xor $t3, $5, $zero
subu $s7, $t6, $25
srl $a1, $27, 2
nor $29, $15, $s0
slt $s3, $k1, $31
nor $gp, $s1, $14
ori $t8, $t4, 9025
lui $k0, 864
ori $15, $k1, 14078
xori $22, $t9, 27449
xor $6, $28, $20
srav $1, $ra, $zero
srl $zero, $29, 11
xori $t2, $a2, 1844
mfhi $s7
mflo $t4
ori $12, $4, 11205
slti $a3, $9, 21316
andi $s2, $28, 17723
ori $11, $s4, 19356
mfhi $21
mflo $t6
addi $a0, $13, 26139
mfhi $17
xori $t5, $k1, 2160
sllv $21, $s6, $13
slti $27, $1, 5900
xori $a2, $t5, 23104
multu $fp, $t7
multu $19, $18
slt $t6, $16, $29
or $31, $s4, $t9
div $9, $6
sltu $t6, $10, $fp
addi $8, $1, 17730
mfhi $17
srlv $12, $27, $s0
sll $17, $t3, 25
xor $t9, $t4, $17
xori $1, $14, 25862
div $s0, $22
ori $15, $19, 27301
ori $12, $s3, 10387
srl $9, $0, 20
subu $28, $19, $31
lui $0, 3249
sltu $ra, $sp, $t0
sltiu $27, $10, 10149
mfhi $k0
mfhi $t5
srl $10, $26, 2
nor $gp, $3, $16
slt $t4, $13, $t2
xor $28, $a0, $a2
multu $13, $13
and $at, $15, $13
slti $s4, $18, 15585
sltu $t6, $v1, $s5
addu $v1, $0, $12
ori $at, $1, 22337
sltiu $sp, $zero, 10096
divu $19, $24
sllv $t7, $6, $15
and $2, $t6, $s6
ori $s5, $s5, 19997
and $gp, $2, $k1
