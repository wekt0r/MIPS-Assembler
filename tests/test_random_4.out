.text
00000000	361400B9	ori	$s4,$s0,185
00000004	01CB4806	srlv	$9,$t3,$14
00000008	00084702	srl	$t0,$8,28
0000000c	0156D022	sub	$k0,$t2,$s6
00000010	0006BB43	sra	$s7,$6,13
00000014	030F6023	subu	$12,$24,$15
00000018	00BA4022	sub	$8,$5,$k0
0000001c	0000E810	mfhi	$29
00000020	01D8A02A	slt	$s4,$t6,$t8
00000024	00144000	sll	$t0,$20,0
00000028	00A3C821	addu	$25,$5,$3
0000002c	27CA6610	addiu	$t2,$fp,26128
00000030	2B8F6A01	slti	$15,$gp,27137
00000034	0000B010	mfhi	$22
00000038	00883827	nor	$7,$a0,$8
0000003c	0266A807	srav	$21,$6,$s3
00000040	2DD91EE5	sltiu	$25,$t6,7909
00000044	02361824	and	$3,$s1,$s6
00000048	004D1023	subu	$2,$v0,$13
0000004c	0000F010	mfhi	$30
00000050	00384804	sllv	$t1,$t8,$at
00000054	361A7164	ori	$k0,$s0,29028
00000058	385E3FB8	xori	$30,$v0,16312
0000005c	030A9006	srlv	$18,$t2,$24
00000060	000D4AC2	srl	$9,$t5,11
00000064	342F42EA	ori	$15,$at,17130
00000068	3C0A34B8	lui	$t2,13496
0000006c	0014A142	srl	$20,$20,5
00000070	03FCC020	add	$t8,$ra,$gp
00000074	0010D840	sll	$k1,$16,1
00000078	010CE821	addu	$29,$8,$t4
0000007c	017BF82B	sltu	$ra,$11,$27
00000080	003AF807	srav	$ra,$k0,$at
00000084	0015C143	sra	$t8,$s5,5
00000088	02E2001B	divu	$23,$2
0000008c	0000A812	mflo	$s5
00000090	00510019	multu	$v0,$17
00000094	28BC06E9	slti	$28,$5,1769
00000098	369C1DB5	ori	$gp,$20,7605
0000009c	0118D807	srav	$k1,$t8,$t0
000000a0	020F1026	xor	$v0,$s0,$t7
000000a4	00007812	mflo	$t7
000000a8	02A0902A	slt	$18,$s5,$0
000000ac	25F55518	addiu	$21,$15,21784
000000b0	0359001B	divu	$k0,$25
000000b4	00C1D807	srav	$k1,$at,$6
000000b8	030FC807	srav	$t9,$t7,$t8
000000bc	00003543	sra	$6,$0,21
000000c0	03E4482A	slt	$9,$31,$4
000000c4	00FA0018	mult	$a3,$26
000000c8	25A25F78	addiu	$v0,$t5,24440
000000cc	034D6823	subu	$13,$26,$t5
000000d0	0000F812	mflo	$ra
000000d4	0146902A	slt	$18,$t2,$6
000000d8	31AA1896	andi	$10,$13,6294
000000dc	00058DC0	sll	$s1,$5,23
000000e0	005CE006	srlv	$28,$28,$v0
000000e4	00030142	srl	$0,$v1,5
000000e8	0102C021	addu	$t8,$8,$v0
000000ec	02470019	multu	$18,$7
000000f0	00009810	mfhi	$s3
000000f4	0044A82A	slt	$21,$2,$a0
000000f8	03BC5822	sub	$t3,$29,$gp
000000fc	012E0018	mult	$t1,$14
00000100	001A5980	sll	$11,$26,6
00000104	26EE1537	addiu	$14,$s7,5431
00000108	013A4020	add	$t0,$t1,$k0
0000010c	0225E006	srlv	$28,$a1,$s1
00000110	02780019	multu	$19,$t8
00000114	0000E012	mflo	$gp
00000118	00C17027	nor	$14,$6,$at
0000011c	3C022ECA	lui	$v0,11978
00000120	00A05826	xor	$t3,$5,$zero
00000124	01D9B823	subu	$s7,$t6,$25
00000128	001B2882	srl	$a1,$27,2
0000012c	01F0E827	nor	$29,$15,$s0
00000130	037F982A	slt	$s3,$k1,$31
00000134	022EE027	nor	$gp,$s1,$14
00000138	35982341	ori	$t8,$t4,9025
0000013c	3C1A0360	lui	$k0,864
00000140	376F36FE	ori	$15,$k1,14078
00000144	3B366B39	xori	$22,$t9,27449
00000148	03943026	xor	$6,$28,$20
0000014c	001F0807	srav	$1,$ra,$zero
00000150	001D02C2	srl	$zero,$29,11
00000154	38CA0734	xori	$t2,$a2,1844
00000158	0000B810	mfhi	$s7
0000015c	00006012	mflo	$t4
00000160	348C2BC5	ori	$12,$4,11205
00000164	29275344	slti	$a3,$9,21316
00000168	3392453B	andi	$s2,$28,17723
0000016c	368B4B9C	ori	$11,$s4,19356
00000170	0000A810	mfhi	$21
00000174	00007012	mflo	$t6
00000178	21A4661B	addi	$a0,$13,26139
0000017c	00008810	mfhi	$17
00000180	3B6D0870	xori	$t5,$k1,2160
00000184	01B6A804	sllv	$21,$s6,$13
00000188	283B170C	slti	$27,$1,5900
0000018c	39A65A40	xori	$a2,$t5,23104
00000190	03CF0019	multu	$fp,$t7
00000194	02720019	multu	$19,$18
00000198	021D702A	slt	$t6,$16,$29
0000019c	0299F825	or	$31,$s4,$t9
000001a0	0126001A	div	$9,$6
000001a4	015E702B	sltu	$t6,$10,$fp
000001a8	20284542	addi	$8,$1,17730
000001ac	00008810	mfhi	$17
000001b0	021B6006	srlv	$12,$27,$s0
000001b4	000B8E40	sll	$17,$t3,25
000001b8	0191C826	xor	$t9,$t4,$17
000001bc	39C16506	xori	$1,$14,25862
000001c0	0216001A	div	$s0,$22
000001c4	366F6AA5	ori	$15,$19,27301
000001c8	366C2893	ori	$12,$s3,10387
000001cc	00004D02	srl	$9,$0,20
000001d0	027FE023	subu	$28,$19,$31
000001d4	3C000CB1	lui	$0,3249
000001d8	03A8F82B	sltu	$ra,$sp,$t0
000001dc	2D5B27A5	sltiu	$27,$10,10149
000001e0	0000D010	mfhi	$k0
000001e4	00006810	mfhi	$t5
000001e8	001A5082	srl	$10,$26,2
000001ec	0070E027	nor	$gp,$3,$16
000001f0	01AA602A	slt	$t4,$13,$t2
000001f4	0086E026	xor	$28,$a0,$a2
000001f8	01AD0019	multu	$13,$13
000001fc	01ED0824	and	$at,$15,$13
00000200	2A543CE1	slti	$s4,$18,15585
00000204	0075702B	sltu	$t6,$v1,$s5
00000208	000C1821	addu	$v1,$0,$12
0000020c	34215741	ori	$at,$1,22337
00000210	2C1D2770	sltiu	$sp,$zero,10096
00000214	0278001B	divu	$19,$24
00000218	01E67804	sllv	$t7,$6,$15
0000021c	01D61024	and	$2,$t6,$s6
00000220	36B54E1D	ori	$s5,$s5,19997
00000224	005BE024	and	$gp,$2,$k1
